### DISCUSSION TITLE
[INFO] [fetch] [2022-06-25T13:55:13.276Z] Please upgrade your Copilot extension to continue using this service.

### DISCUSSION THREAD

**Discussion by cdvzanmyad** [community/community#19423] [Projects and Issues]
![image](https://user-images.githubusercontent.com/108179954/175776609-ea8bc58e-d595-4df8-9201-45c375c84167.png)
Help!

**Comment by melvincwng** [comment]
@cdvzanmyad, try the following:

1. Update VSCode to the latest version. 
2. Update Github Copilot to the latest version.

If the above helps, please 'Mark as answer' so others are able to see the solution.

References:
- [StackOverFlow](https://stackoverflow.com/questions/72311242/please-upgrade-your-copilot-extension-to-continue-using-this-service)
- [Github Community Discussion #15684](https://github.com/github-community/community/discussions/15684)
